.github/workflows/osv-scanner-scan.yml

```
@@ -6,6 +6,9 @@ name: OSV-Scanner Scan
 on:
   schedule:
   - cron: "12 12 * * 1"
+  # testing
+  pull_request:
+    branches: [main]
```

Use list format instead.
```suggestion
    branches:
    - main
```